mod-di-converter-storage-server/src/main/resources/templates/db_scripts/data-migration/remove_accepted_values_instance_orders.sql

```diff
@@ -0,0 +1,28 @@
+-- Create function that removes jsonb property recursive
+CREATE OR REPLACE FUNCTION remove_key_recursive(data jsonb, target_key text)
+RETURNS jsonb LANGUAGE plpgsql AS $$
+BEGIN
+    IF jsonb_typeof(data) = 'object' THEN
+        RETURN (
+            SELECT jsonb_object_agg(k, remove_key_recursive(v, target_key))
+            FROM jsonb_each(data) AS t(k, v)
+            WHERE k <> target_key
+        );
+    ELSIF jsonb_typeof(data) = 'array' THEN
+        RETURN (
+            SELECT jsonb_agg(remove_key_recursive(elem, target_key))
+            FROM jsonb_array_elements(data) AS elem
+        );
+    ELSE
+        RETURN data;
+    END IF;
+END;
+$$;
+
+-- Remove 'acceptedValues' property from 'INSTANCE', 'HOLDINGS', 'ITEM' and 'ORDER' mapping profile
+UPDATE ${myuniversity}_${mymodule}.mapping_profiles
+SET jsonb = remove_key_recursive(jsonb, 'acceptedValues')
```

Comment:
perhaps we might consider passing only the "mappingDetails" array to the remove_key_recursive() function to reduce the number of iterations through each mapping profile.